[2024-12-05T10:26:33.399+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T10:26:33.407+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T10:26:33.408+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T10:26:33.408+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-12-05T10:26:33.408+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T10:26:33.420+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): data-cleaning> on 2024-11-30 09:20:00+00:00
[2024-12-05T10:26:33.425+0000] {standard_task_runner.py:52} INFO - Started process 2369 to run task
[2024-12-05T10:26:33.427+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'P2M3', 'data-cleaning', 'scheduled__2024-11-30T09:20:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_alridho_DAG.py', '--cfg-path', '/tmp/tmpg4_e10ga', '--error-file', '/tmp/tmps24vda2p']
[2024-12-05T10:26:33.428+0000] {standard_task_runner.py:80} INFO - Job 17: Subtask data-cleaning
[2024-12-05T10:26:33.524+0000] {task_command.py:371} INFO - Running <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [running]> on host 91c538a0d2ed
[2024-12-05T10:26:33.636+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alridho
AIRFLOW_CTX_DAG_ID=P2M3
AIRFLOW_CTX_TASK_ID=data-cleaning
AIRFLOW_CTX_EXECUTION_DATE=2024-11-30T09:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-30T09:20:00+00:00
[2024-12-05T10:26:33.993+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-12-05T10:26:34.025+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=P2M3, task_id=data-cleaning, execution_date=20241130T092000, start_date=20241205T102633, end_date=20241205T102634
[2024-12-05T10:26:34.083+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-05T10:26:34.160+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T11:34:57.311+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T11:34:57.319+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T11:34:57.320+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T11:34:57.320+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-12-05T11:34:57.320+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T11:34:57.334+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): data-cleaning> on 2024-11-30 09:20:00+00:00
[2024-12-05T11:34:57.340+0000] {standard_task_runner.py:52} INFO - Started process 407 to run task
[2024-12-05T11:34:57.343+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'P2M3', 'data-cleaning', 'scheduled__2024-11-30T09:20:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_alridho_DAG.py', '--cfg-path', '/tmp/tmpzir37glp', '--error-file', '/tmp/tmposkjynbf']
[2024-12-05T11:34:57.343+0000] {standard_task_runner.py:80} INFO - Job 31: Subtask data-cleaning
[2024-12-05T11:34:57.485+0000] {task_command.py:371} INFO - Running <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [running]> on host 9b483ed80620
[2024-12-05T11:34:57.595+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alridho
AIRFLOW_CTX_DAG_ID=P2M3
AIRFLOW_CTX_TASK_ID=data-cleaning
AIRFLOW_CTX_EXECUTION_DATE=2024-11-30T09:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-30T09:20:00+00:00
[2024-12-05T11:34:57.625+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-12-05T11:34:57.648+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=P2M3, task_id=data-cleaning, execution_date=20241130T092000, start_date=20241205T113457, end_date=20241205T113457
[2024-12-05T11:34:57.675+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-05T11:34:57.727+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T11:45:27.051+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T11:45:27.058+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T11:45:27.058+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T11:45:27.058+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-12-05T11:45:27.059+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T11:45:27.070+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): data-cleaning> on 2024-11-30 09:20:00+00:00
[2024-12-05T11:45:27.077+0000] {standard_task_runner.py:52} INFO - Started process 610 to run task
[2024-12-05T11:45:27.079+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'P2M3', 'data-cleaning', 'scheduled__2024-11-30T09:20:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_alridho_DAG.py', '--cfg-path', '/tmp/tmp9u4u5sfi', '--error-file', '/tmp/tmpk7ekmqaw']
[2024-12-05T11:45:27.080+0000] {standard_task_runner.py:80} INFO - Job 42: Subtask data-cleaning
[2024-12-05T11:45:27.182+0000] {task_command.py:371} INFO - Running <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [running]> on host 9b483ed80620
[2024-12-05T11:45:27.283+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alridho
AIRFLOW_CTX_DAG_ID=P2M3
AIRFLOW_CTX_TASK_ID=data-cleaning
AIRFLOW_CTX_EXECUTION_DATE=2024-11-30T09:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-30T09:20:00+00:00
[2024-12-05T11:45:27.313+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-12-05T11:45:27.337+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=P2M3, task_id=data-cleaning, execution_date=20241130T092000, start_date=20241205T114527, end_date=20241205T114527
[2024-12-05T11:45:27.371+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-05T11:45:27.425+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T11:49:45.328+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T11:49:45.335+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T11:49:45.335+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T11:49:45.336+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-12-05T11:49:45.336+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T11:49:45.347+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): data-cleaning> on 2024-11-30 09:20:00+00:00
[2024-12-05T11:49:45.352+0000] {standard_task_runner.py:52} INFO - Started process 708 to run task
[2024-12-05T11:49:45.354+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'P2M3', 'data-cleaning', 'scheduled__2024-11-30T09:20:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_alridho_DAG.py', '--cfg-path', '/tmp/tmpk6vpatlv', '--error-file', '/tmp/tmpn7tx9wld']
[2024-12-05T11:49:45.355+0000] {standard_task_runner.py:80} INFO - Job 50: Subtask data-cleaning
[2024-12-05T11:49:45.442+0000] {task_command.py:371} INFO - Running <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [running]> on host 9b483ed80620
[2024-12-05T11:49:45.536+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alridho
AIRFLOW_CTX_DAG_ID=P2M3
AIRFLOW_CTX_TASK_ID=data-cleaning
AIRFLOW_CTX_EXECUTION_DATE=2024-11-30T09:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-30T09:20:00+00:00
[2024-12-05T11:49:45.569+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-12-05T11:49:45.592+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=P2M3, task_id=data-cleaning, execution_date=20241130T092000, start_date=20241205T114945, end_date=20241205T114945
[2024-12-05T11:49:45.646+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-05T11:49:45.698+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T12:01:16.647+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T12:01:16.656+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T12:01:16.656+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T12:01:16.656+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-12-05T12:01:16.656+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T12:01:16.668+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): data-cleaning> on 2024-11-30 09:20:00+00:00
[2024-12-05T12:01:16.774+0000] {standard_task_runner.py:52} INFO - Started process 920 to run task
[2024-12-05T12:01:16.777+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'P2M3', 'data-cleaning', 'scheduled__2024-11-30T09:20:00+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_alridho_DAG.py', '--cfg-path', '/tmp/tmpl5y3l77l', '--error-file', '/tmp/tmpng1mobjj']
[2024-12-05T12:01:16.777+0000] {standard_task_runner.py:80} INFO - Job 58: Subtask data-cleaning
[2024-12-05T12:01:16.920+0000] {task_command.py:371} INFO - Running <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [running]> on host 9b483ed80620
[2024-12-05T12:01:17.021+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alridho
AIRFLOW_CTX_DAG_ID=P2M3
AIRFLOW_CTX_TASK_ID=data-cleaning
AIRFLOW_CTX_EXECUTION_DATE=2024-11-30T09:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-30T09:20:00+00:00
[2024-12-05T12:01:17.051+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-12-05T12:01:17.075+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=P2M3, task_id=data-cleaning, execution_date=20241130T092000, start_date=20241205T120116, end_date=20241205T120117
[2024-12-05T12:01:17.109+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-05T12:01:17.162+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-05T12:05:08.139+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T12:05:08.147+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T12:05:08.148+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T12:05:08.148+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-12-05T12:05:08.148+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T12:05:08.162+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): data-cleaning> on 2024-11-30 09:20:00+00:00
[2024-12-05T12:05:08.169+0000] {standard_task_runner.py:52} INFO - Started process 1007 to run task
[2024-12-05T12:05:08.171+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'P2M3', 'data-cleaning', 'scheduled__2024-11-30T09:20:00+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_alridho_DAG.py', '--cfg-path', '/tmp/tmp3sb0mk0y', '--error-file', '/tmp/tmpsj_zqrxt']
[2024-12-05T12:05:08.172+0000] {standard_task_runner.py:80} INFO - Job 65: Subtask data-cleaning
[2024-12-05T12:05:08.268+0000] {task_command.py:371} INFO - Running <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [running]> on host 9b483ed80620
[2024-12-05T12:05:08.358+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alridho
AIRFLOW_CTX_DAG_ID=P2M3
AIRFLOW_CTX_TASK_ID=data-cleaning
AIRFLOW_CTX_EXECUTION_DATE=2024-11-30T09:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-30T09:20:00+00:00
[2024-12-05T12:05:08.387+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-12-05T12:05:08.412+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=P2M3, task_id=data-cleaning, execution_date=20241130T092000, start_date=20241205T120508, end_date=20241205T120508
[2024-12-05T12:05:08.464+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-05T12:05:08.531+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T12:09:34.622+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T12:09:34.629+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T12:09:34.630+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T12:09:34.630+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-12-05T12:09:34.630+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T12:09:34.642+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): data-cleaning> on 2024-11-30 09:20:00+00:00
[2024-12-05T12:09:34.647+0000] {standard_task_runner.py:52} INFO - Started process 1092 to run task
[2024-12-05T12:09:34.649+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'P2M3', 'data-cleaning', 'scheduled__2024-11-30T09:20:00+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_alridho_DAG.py', '--cfg-path', '/tmp/tmpvxiw29u_', '--error-file', '/tmp/tmpmfofq974']
[2024-12-05T12:09:34.651+0000] {standard_task_runner.py:80} INFO - Job 74: Subtask data-cleaning
[2024-12-05T12:09:34.742+0000] {task_command.py:371} INFO - Running <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [running]> on host 9b483ed80620
[2024-12-05T12:09:34.845+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alridho
AIRFLOW_CTX_DAG_ID=P2M3
AIRFLOW_CTX_TASK_ID=data-cleaning
AIRFLOW_CTX_EXECUTION_DATE=2024-11-30T09:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-30T09:20:00+00:00
[2024-12-05T12:09:34.877+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-12-05T12:09:34.900+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=P2M3, task_id=data-cleaning, execution_date=20241130T092000, start_date=20241205T120934, end_date=20241205T120934
[2024-12-05T12:09:34.942+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-05T12:09:34.999+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T12:19:36.509+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T12:19:36.517+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [queued]>
[2024-12-05T12:19:36.517+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T12:19:36.518+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-12-05T12:19:36.518+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T12:19:36.528+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): data-cleaning> on 2024-11-30 09:20:00+00:00
[2024-12-05T12:19:36.533+0000] {standard_task_runner.py:52} INFO - Started process 227 to run task
[2024-12-05T12:19:36.535+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'P2M3', 'data-cleaning', 'scheduled__2024-11-30T09:20:00+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_alridho_DAG.py', '--cfg-path', '/tmp/tmpdvuvepbs', '--error-file', '/tmp/tmp2re1yg_w']
[2024-12-05T12:19:36.537+0000] {standard_task_runner.py:80} INFO - Job 84: Subtask data-cleaning
[2024-12-05T12:19:36.624+0000] {task_command.py:371} INFO - Running <TaskInstance: P2M3.data-cleaning scheduled__2024-11-30T09:20:00+00:00 [running]> on host f1a3a7cf14eb
[2024-12-05T12:19:36.790+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alridho
AIRFLOW_CTX_DAG_ID=P2M3
AIRFLOW_CTX_TASK_ID=data-cleaning
AIRFLOW_CTX_EXECUTION_DATE=2024-11-30T09:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-30T09:20:00+00:00
[2024-12-05T12:19:36.872+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-12-05T12:19:36.902+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=P2M3, task_id=data-cleaning, execution_date=20241130T092000, start_date=20241205T121936, end_date=20241205T121936
[2024-12-05T12:19:36.948+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-05T12:19:37.025+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
